e/eclipse-che/che-devfile-registry/build-info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "che-devfile-registry",
+    "github_url":"https://github.com/eclipse-che/che-devfile-registry.git",
+    "version": "7.63.0",
+    "default_branch": "main",
+    "package_dir": "e/eclipse-che/che-devfile-registry", 
```

Comment:
@shreyakajbaje  `package_dir` should be `c/che-devfile-registry` here

Reply:
I noticed this structure is already there but will not help for currency need. So lets create folder like e/eclipse-che-devfile-registry.  You can add new script in build_info, also copy older script also to same.

Reply:
@shreyakajbaje we will need to delete existing folder so we make sure to have all scripts in one folder and not create duplicates.
Can you please delete `eclipse-che/che-devfile-registry`